### PR TITLE
Refer to sessions correctly, instead of devices.

### LINKED ISSUE #69: Reference 'sessions' not 'devices'
## Overview
There is some expected overlap between the phrases 'sessions' and 'devices' when looking at this feature. We need to use both terms to paint a clear picture of what's happening. Upon reviewing the current UI, there has been some discussion on tweaking the use of these terms slightly.

At a high-level, it's suggested that we refer to the information and frame it as a 'session', not a 'device' That's also more technically correct given it's the session data that the module is tracking - there's no 'device_ID' for example.

* In the main component, we currently refer to the information as "Authenticated devices" - which makes less sense when it's possible to have multiple sessions against a single device, see:
https://invis.io/ANZZEC4XUTQ#/439163639_CMS_-_Login_-_Session_Management_-_Remove_A_Device_-_List

## Acceptance Criteria
- [ ]  The label in the Profile admin is updated
- [ ] Review existing docs and screenshots that may reference this UI, make necessary changes

## PRs
- [ ] https://github.com/silverstripe/silverstripe-session-manager/pull/71

## Notes
-

### PR DESCRIPTION
**Pull request by bergice**
Resolves https://github.com/silverstripe/silverstripe-session-manager/issues/69

### CODE REVIEW

File: src/Extensions/MemberExtension.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function updateCMSFields(FieldList $fields)
             'Root.Main',
             SessionManagerField::create(
                 'LoginSessions',
-                _t(__CLASS__ . '.DEVICES', 'Authenticated devices'),
+                _t(__CLASS__ . '.DEVICES', 'Login sessions'),
```
Comment (maxime-rainville) [inline src/Extensions/MemberExtension.php]: The key should match the actual source string.

```suggestion
                _t(__CLASS__ . '.LOGIN_SESSIONS', 'Login sessions'),
```

Alternatively, you could do this and update the `singular_name` on Login Session to `Login session`.
```suggestion
                LoginSession::singleton()->plural_name(),
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Run `sake dev/tasks/i18nTextCollectorTask` so all the translation strings are picked up.

File: src/Extensions/MemberExtension.php (hunk 1 above)
Comment (bergice) [reply src/Extensions/MemberExtension.php]: > Run sake dev/tasks/i18nTextCollectorTask so all the translation strings are picked up.

That's done now.

File: images/lang/en.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ en:
     MENUTITLE: SilverStripe\SessionManager\Controllers\LoginSessionController
     REMOVE_FAILURE: 'Something went wrong.'
     REMOVE_PERMISSION: 'You do not have permission to delete this record.'
-    REMOVE_SUCCESS: 'Successfully logged out of device.'
+    REMOVE_SUCCESS: 'Successfully removed session.'
```
Comment (maxime-rainville) [inline images/lang/en.yml:5]: There shouldn't be a `lang` folder under images.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Why is there a `lang` folder under `images` and why are we removing that `images/cms.png` file.

Otherwise, everything else looks good :ok_hand:

File: images/lang/en.yml (hunk 2 above)
Comment (bergice) [reply images/lang/en.yml:5]: No idea why, but I've changed it.

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**